### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by lvnhmd**
Just to validate that we are on the right track

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -1,46 +1,107 @@
...
+          console.log('SEND TO TWO PEOPLE');
+          // I still have the basci user logged in - I need to notify him as well
+          tos.push({
+            name: `${json.meta.user.profile.firstName} ${
+              json.meta.user.profile.lastName
+            }`,
+            to: 'swagbag@swagbag.club', //json.meta.user.profile.email
+            template: 'notification_task_change'
+          });
+          // how do I find the NTCO ?
```
Comment (joefitter) [inline lib/app.js]: query the database. You'll be able to find out the establishmentId from the task then you'll need to query roles table where establishmentId === task.data.establishmentId && role.type === 'ntco'

File: config.js [hunk 2]
```diff
@@ -1 +1,34 @@
-module.exports = {};
+module.exports = {
+  port: process.env.PORT || 8087,
+  db: {
+    database: process.env.DATABASE_NAME || 'asl',
+    host: process.env.DATABASE_HOST || 'localhost',
+    user: process.env.DATABASE_USERNAME || 'postgres'
+  },
+  emailer: process.env.EMAILER_SERVICE,
+  publicUrl: process.env.PUBLIC_UI,
```
Comment (lennym) [inline config.js]: We don't have any situations where we send email to internal users yet do we?

File: lib/errors/index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  NotFoundError: require('./not-found'),
+  BadRequestError: require('./bad-request')
+};
```
Comment (lennym) [inline lib/errors/index.js]: Use the errors directly from `@asl/service/errors`

File: config.js (hunk 2 above)
Comment (joefitter) [reply config.js]: no not yet

File: lib/errors/index.js (hunk 3 above)
Comment (joefitter) [reply lib/errors/index.js]: ooh, shiny

**Pull request merged**